src/tigris.ts

```diff
@@ -274,6 +276,34 @@ export class Tigris {
 			});
 		});
 	}
+
+	/**
+	 * Automatically provision Databases and Collections based on the directories
+	 * and {@link TigrisSchema} definitions in file system
+	 *
+	 * @param absoluteSchemaPath - Directory location in file system from root
+	 * directory (/) to load schemas from
+	 */
+	public async registerSchemas(absoluteSchemaPath: string) {
+		const manifest: TigrisManifest = loadTigrisManifest(absoluteSchemaPath);
+
+		for (const dbManifest of manifest) {
+			// create DB
+			const tigrisDb = await this.createDatabaseIfNotExists(dbManifest.dbName);
+			Log.event(`Created database: ${dbManifest.dbName}`);
```

Comment:
add a check if there is no collections associated with this db - we can skip creating db. it could be a unwanted dir.